### ISSUE TITLE
Pairing over USB

### ISSUE THREAD

**Issue by s0r00t** [rootkiwi/an2linuxserver#8]
I'm not sure if that's possible without a custom kernel, but that would be great : 

- When your computer doesn't have Bluetooth
- When you don't want notifications to be sent through WiFi/mobile

**Comment by rootkiwi** [comment]
Yes that would be great, I know almost nothing about USB but definitely something I will look up some day

**Comment by arkadiusz-wieczorek** [comment]
I think that through USB you can also pair phone with computer for BT/WiFi/Mobile.

**Comment by BLACKWHITEHACKERSOFIICE** [comment]
Iam indonesian iam hack hackers

**Comment by BLACKWHITEHACKERSOFIICE** [comment]
Do you from ?

**Comment by BLACKWHITEHACKERSOFIICE** [comment]
My name is joni to the sumatra selatan indonesia n
